Makefile

```diff
@@ -5,6 +5,11 @@ ifeq ($(UNAME_M), x86_64)
 else
 	ARCH=$(UNAME_M)
 endif
+ifeq ($(ARCH), amd64)
+	DOCKERFILE = Dockerfile
```

Comment:
maybe easier to rename Dockerfile to Dockerfile.amd64 and remove this logic and use $(ARCH) ?

Reply:
done